dataland-backend/src/main/kotlin/org/dataland/datalandbackend/api/CompanyApi.kt

```diff
@@ -67,20 +67,23 @@ interface CompanyApi {
      * A method to retrieve specific companies identified by different filters
      * If the filters are not set, all companies in the data store are returned.
      * @param searchString string used for substring matching
-     * @param onlyCompanyNames boolean determining if the search should be solely against the companyNames
      * @param dataTypes If set & non-empty,
      * this function only returns companies that have data for the specified dataTypes
      * @param countryCodes If set & non-empty,
      * this function only returns companies that have a country code contained in the set
      * @param sectors If set & non-empty, this function only returns companies that belong to a sector in the set
+     * @param onlyCompanyNames boolean determining if the search should be solely against the companyNames
+     * @param onlyCurrentUserAsUploader boolean determining if the search should only find dataset metainfos of datasets
```

Comment:
Line 76. onlyCurrentUserAsUploader and it's description seem to be missmatched

Reply:
Sorry to be bothering you with this again, but shouldn't it be "find dataset metainfos belonging to the current user" instead of "find dataset metainfos of datasets"?

Reply:
I checked my code files. This is not uptodate. It should be "find companies with datasets"

Reply:
Ok